MediaWikiTelegramBot.js

```diff
@@ -284,7 +293,38 @@ function showCurrentMwMessage(userID) {
 function advanceMwMessage(userID) {
     const user = getUser(userID);
 
+    // Make sure there is another message to translate
+    if( user.currentMwMessageIndex + 1 >= user.loadedMwMessages.length ) {
```

Comment:
A very tiny nitpick: There must be a space after if, and no spaces inside the parentheses.

There used to be spaces inside the parentheses in the earlier version of the code, but I adapted the coding conventions of other Wikimedia Node.js services, and I'm enforcing them with eslint.

If you're using the Atom editor, you can integrate automatic checking while editing using the linter-eslint package.